capa/features/extractors/dnfile/helpers.py

```diff
@@ -299,22 +327,79 @@ def get_dotnet_unmanaged_imports(pe: dnfile.dnPE) -> Iterator[DnUnmanagedMethod]
         # like kernel32.CreateFileA
         yield DnUnmanagedMethod(token, module, method)
 
-
+def get_nested_class_table(pe):
```

Comment:
Can we give this function a more descriptive name and add a comment describing what the functions does?  This function appears to be building out an index for the `EnclosingClass` columns versus simply return the `NestedClass` table.

Reply:
Renamed to `enclosing_and_nested_classes_index_table` and added Python type annotation for the better description.